### DISCUSSION TITLE
could you add a makefile directly into the project instead of autogen.sh , thanks.

### DISCUSSION THREAD

**Discussion by owenliang** [protobuf-c/protobuf-c#668] [Q&A] [answered]
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4 ${ACLOCAL_FLAGS}
aclocal:configure.ac:21: warning: macro`AM_SILENT_RULES' not found in library
aclocal:configure.ac:30: warning: macro `AM_COND_IF' not found in library
configure.ac:1: error: Autoconf version 2.64 or higher is required
configure.ac:1: the top level
autom4te: /usr/bin/m4 failed with exit status: 63
aclocal: autom4te failed with exit status: 63
autoreconf: aclocal failed with exit status: 63

**Comment by alex85k** [comment]
You can try CMake scripts from this fork: https://github.com/alex85k/protobuf-c
(`cd build-cmake; cmake .. ; make install`) 
(created for Windows, may have some bugs. You can replace the .c and .h files with unmodified versions)

**Comment by edmonds** [comment, marked as answer]
Hello,

We do not keep any generated files in the repository, and in any case the `Makefile` is build specific. I recommend using the latest tarball release from https://github.com/protobuf-c/protobuf-c/releases and running `./configure`.
